langchain/src/vectorstores/cassandra.ts

```diff
@@ -313,15 +342,30 @@ export class CassandraStore extends VectorStore {
     cql = `CREATE CUSTOM INDEX IF NOT EXISTS idx_vector_${this.table}
            ON ${this.keyspace}.${
       this.table
-    }(vector) USING 'StorageAttachedIndex' WITH OPTIONS = {'similarity_function': '${this.vectorType.toUpperCase()}'};`;
+    }(vector) USING 'StorageAttachedIndex' WITH OPTIONS = {'similarity_function': '${this.vectorType.toLowerCase()}'};`;
```

Comment:
Is this breaking?

Reply:
this is case insensitive in Cassandra, the docs indicate lowercase so just making consistent.